src/main/java/com/example/bookstore/repository/BookRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.example.bookstore.repository;
+
+import com.example.bookstore.model.Book;
+import java.util.List;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface BookRepository {
+    Book save(Book book);
+
+    List<Book> findAll();
+}
```

Comment:
@Repository не потрібна тут. 

```suggestion
public interface BookRepository {
    Book save(Book book);

    List<Book> findAll();
}
```